dokumentpakking/src/main/java/no/difi/meldingsutveksling/dokumentpakking/service/CreateSBD.java

```diff
@@ -28,18 +29,33 @@
 
     public EduDocument createSBD(Organisasjonsnummer avsender, Organisasjonsnummer mottaker, Object payload, String conversationId, String type, String journalPostId) {
 		EduDocument doc = new EduDocument();
-		doc.setStandardBusinessDocumentHeader(createHeader(avsender, mottaker, conversationId, type, journalPostId));
+		doc.setStandardBusinessDocumentHeader(createHeader(avsender, mottaker, conversationId, type, journalPostId, null));
```

Comment:
Burde kanskje overlastet create header for denne ekstra parameteren i stedet for å sende med null der den ikke finnes.